### PR TITLE
To do list basic set up

### PR DESCRIPTION
**Pull request by shayan1234554321**
Basic setup of html , css and js is added.
Js file changes :-
- Loads list items on window content load
- Loads items according to item index

HTML file changes :-
- Added header
- Added form
- Added list container
- Added remove button

CSS file changes :-
Added styling for all the content added in html

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,8 +1,147 @@
...
+  background-color: rgba(255, 255, 255, 0);
+  outline: none;
+  color: rgb(118, 175, 255);
+  border: 2px solid rgb(11, 112, 255);
+  border-radius: 7px;
+  font-weight: 600;
+  font-size: 14px;
+}
+
+ul {
```
Comment (AmaduKamara) [inline src/style.css:93]: Good job so far, the style of your app looks nice but I will highly recommend you follow the design given in the video.

- [ ] For this project, the to-do list app, you're required to style the app to look as close as the one in the video. Kindly modify your styles for the entire app to match the design given in the project requirement 👍🏽

**The Required Design**
Screenshot 2023-02-22 at 12 18 24 AM

**Your Design**
Screenshot 2023-02-22 at 12 19 25 AM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @shayan1234554321,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly
✅ Linter checks passed 
✅ Used Webpack correctly

## Required Changes ♻️
_Check the comments under the review._

- [ ] Let's try to follow the requirements and design our app the expected way. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/style.css (hunk 1 above)
Comment (shayan1234554321) [reply src/style.css:93]: Sure I will, But isn't that design boring? I thought a custom design would be appreciated!
But no worries, I will do that design

**Review by PinkMoon25 — APPROVED** [review]
Hi @shayan1234554321 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Webpack used :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**